tests/fixtures/saas_fixtures.py

```diff
@@ -40,38 +45,63 @@ def load_config(filename: str) -> Dict:
 
 
 @pytest.fixture
-def example_saas_configs() -> Dict[str, Dict]:
-    example_saas_configs = {}
-    example_saas_configs["mailchimp"] = load_config(
-        "data/saas/config/mailchimp_config.yml"
+def saas_configs() -> Dict[str, Dict]:
```

Comment:
This was actually a pattern I was hoping we could move away from. It makes more sense to address each config in a fixture individually I think, so we don't need to load in every single one and address it opaquely as a list index.